shippable.resources.yml

```diff
@@ -14,6 +14,6 @@ resources:
   # config for GKE CLI
   - name: gke_cliConfig
     type: cliConfig
-    integration: drship_gke
+    integration: drship_gke    # replace with your Google Cloud integration name
```

Comment:
we need to change the integration name here to the new google cloud integration. Not sure what the name should be so havent' changed till yet